client/my-sites/site/index.jsx

```diff
@@ -16,7 +16,24 @@ module.exports = React.createClass( {
 	displayName: 'Site',
 
 	componentDidMount: function() {
-		debug( 'The Site component is mounted.' );
+		if ( this.props.isSelected ) {
+			this.scrollIntoView();
```

Comment:
I think this logic should be placed in the site-selector component, not in the individual site.